testcontainers/compose.py

```diff
@@ -89,6 +89,19 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.stop()
+        
+    @cached_property
+    def base_docker_compose(self):
+        """
+        Returns the basecommand parts used for the docker compose commands depending on the docker compose api
+
+        Returns
+        -------
+        list[str]
+            The docker compose command parts
+        """
+        return ["docker","compose"] if subprocess.run(["docker", "compose", "--help"], stdout=subprocess.DEVNULL,
+        stderr=subprocess.STDOUT).returncode == 0 else ["docker-compose"]
```

Comment:
I wonder if this should be configurable by the user ? There might be cases where the user prefers `docker-compose` over `docker compose`. Maybe the magic here is good, but as fallback from a user provided setting?

Reply:
That sounds reasonable, maybe we can include a `compose_command` argument in the constructor. If it is `None`, we can use the magic to figure out the right one. Otherwise the provided value will be used as-is.

Reply:
@tillahoffmann Given that we have moved all containers to their directory, should we rebase this PR ? Maybe one of us will have to do it if OP is not around.

Reply:
Accidentally closed this PR and opened #312 which includes the suggested changes